spec/lucky/action_spec.cr

```diff
@@ -99,6 +99,10 @@ describe Lucky::Action do
       Tests::Update.with("test-id").should eq Lucky::RouteHelper.new(:put, "/tests/test-id")
       Tests::Create.path.should eq "/tests"
       Tests::Create.route.should eq Lucky::RouteHelper.new(:post, "/tests")
```

Comment:
Could you add  test for `Tests::Index.with(anchor: "foo")` to test an action that only uses an anchor and no other params?